src/python/bot/tasks/minimize_task.py

```diff
@@ -37,6 +37,8 @@
 from bot.minimizer import js_minimizer
 from bot.minimizer import js_tokenizer
 from bot.minimizer import minimizer
+#from bot.minimizer.antlr_tokenizer import AntlrTokenizer
```

Comment:
Can skip this step. Lets not use the code yet in minimizer, this part can go in next CL, also we don't check in commented code.

Reply:
Done! It makes sense not to check in commented code. Thanks for the explanation